reader/processor/processor.go

```diff
@@ -80,14 +80,15 @@ func ProcessFeedEntries(store *storage.Storage, feed *model.Feed, user *model.Us
 				logger.Error(`[Processor] Unable to crawl this entry: %q => %v`, entry.URL, scraperErr)
 			} else if content != "" {
 				// We replace the entry content only if the scraper doesn't return any error.
-				entry.Content = content
+				// TODO: document change
+				entry.WebContent = content
 			}
 		}
 
 		rewrite.Rewriter(url, entry, feed.RewriteRules)
```

Comment:
This is problematic, I didn't look at how rewriter gets its data - I'll add a commit which fixes this.